### PR TITLE
Updated package to version 1.1.0 [MERGE FIRST]

### PR DESCRIPTION
**Pull request by Debadri-Redapple**
### What Does This PR Do?
Necessary updates for version 1.1.0

### Checklist (if applicable):
[ ] item 1
[ ] item 2
[ ] item 3
[ ] item 4
[ ] item 5

### CODE REVIEW

**Review by noah1013 — APPROVED** [review]
Looks good 👍

**Review by drxeno02 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # TG-MIKROS-SDK-unity-package
 MIKROS is a SaaS product that enrolls developers in an information sharing ecosystem, also known as data pooling, that helps identify better insights about user behavior. Including user spending habits.
+
+## Installation
```
Comment (drxeno02) [inline README.md:4]: 👍

**Comment by drxeno02** [comment]
@Debadri-Redapple @noah1013 If you are happy with this PR, let's get it merged in. I will update `main` afterwards. Thanks!

**Comment by Debadri-Redapple** [comment]
@drxeno02 I don't have access to merge this PR. Its showing me "You’re not authorized to merge this pull request."
Please look into this.

**Comment by drxeno02** [comment]
> @drxeno02 I don't have access to merge this PR. Its showing me "You’re not authorized to merge this pull request." Please look into this.

@Debadri-Redapple That should be corrected now. Let me know.

**Comment by Debadri-Redapple** [comment]
@drxeno02 Yep, was able to merge.

**Pull request merged**